core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts

```diff
@@ -61,13 +63,22 @@ describe('Exploration player page', function() {
   it('should load skill based on its id on url when component is initialized' +
     ' and set angular element content property based on the exploration',
   function() {
+    jasmine.getEnv().allowRespy(true);
```

Comment:
Alternatively, maybe you could create the elements, e.g. like what is done [here](https://github.com/oppia/oppia/blob/develop/core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.spec.ts#L52). Maybe the other reviewers can comment on this as well.